cmd/format-erasure.go

```diff
@@ -890,18 +889,23 @@ func initFormatErasure(ctx context.Context, storageDisks []StorageAPI, setCount,
 	return getFormatErasureInQuorum(formats)
 }
 
+func getDefaultParityBlocks(drive int) int {
+	switch drive {
+	case 4, 5:
+		return 2
+	case 6, 7:
+		return 3
+	default:
```

Comment:
Maybe add something sensible for < 4.

Reply:
We will never have drives less that 4 erasure stripe size, are you suggesting we just add it anyways ?

Reply:
Yes, just as a defensive measure, things that hold true now may change in the future and this is easy to overlook.